services/agents/registry.go

```diff
@@ -53,6 +53,8 @@ const (
 )
 
 var (
+	checkExternalExporterConnectionPMMVersion = version.MustParse("1.15.0")
```

Comment:
`pmmAgentVersion.Less` works weird that our FBs version will return true in this case, so let's use 1.14.99 instead of 1.15.0